<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependencyManagerMain</class>
 <widget class="QMainWindow" name="DependencyManagerMain">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>670</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nyala</family>
    <pointsize>17</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QListView" name="usedListView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListView" name="availableListView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Logs:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="allRightButton">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Resources/all_right.png</normaloff>:/res/Resources/all_right.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightButton">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Resources/right.png</normaloff>:/res/Resources/right.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftButton">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Resources/left.png</normaloff>:/res/Resources/left.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="allLeftButton">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Resources/all_left.png</normaloff>:/res/Resources/all_left.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="generateButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="staticLabel">
        <property name="text">
         <string>Available modules:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="staticLabel_1">
        <property name="text">
         <string>Used modules:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QPushButton" name="configureButton">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_default_settings"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionMinimize"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionGenerate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_me"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar {
height: 1px;
}</string>
   </property>
  </widget>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRebuild">
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="text">
    <string>Save project as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLoad_default_settings">
   <property name="text">
    <string>Start from scratch</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionHelp_me">
   <property name="text">
    <string>Help me</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
